src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+@import "./utils/variables";
+
+* {
```

Comment:
Using the universal selector (*) can lead to performance issues. It would be better to apply the rules to the specific elements you want to style. It's only allowed to use `box-sizing: border-box;` to `*` selector